### ISSUE TITLE
Alphanumeric TEID field

### ISSUE THREAD

**Issue by pstav** [lagopus/lagopus#137]
Hello, 

Tried to inlcude alphanumeric in TEID field of GTP as per standard but it only accepts 4 bytes integer.

Thanks

**Comment by fr6nco** [comment]
@pstav I don't think you can use alphanumeric chars in the TEID field. TEID is always a 4 byte integer, su basically you could translate it to a 4 char ASCII string, however, in the end it's still a 32 bit number.

https://en.wikipedia.org/wiki/GPRS_Tunnelling_Protocol#GTP_version_2
